src/BatteryStateOfChargeService.h

```diff
@@ -19,4 +27,27 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     const double initialStateOfChargePercent_;
 
     // Recommend adding necessary variables here
+    double initialCurrent;
+    double newCurrent = 0;
+    QTime initialTime;
+    QTime newTime;
+
+    double dTime;
+    double aveCurrent;
+    double dAmpHours;
+    double initialampHours;
+    double timeRemaining;
+    double ampsRemaining;
+    double rateofDepletion;
+    double hoursPrime;
+    double minutesPrime;
+    double secondsPrime;
+    double aveCurrentPrime;
+    double currentPrime = 0;
```

Comment:
The `Prime` word is not very meaningful, it would be better to change this to something all people would understand. As it stands, neither Jordan and I do not understand what this variable is supposed to do.